webdriver/tests/classic/new_session/merge.py

```diff
@@ -41,42 +41,57 @@ def test_merge_invalid(new_session, add_browser_capabilities, key, value):
 
 
 def test_merge_platform_name(new_session, add_browser_capabilities, target_platform):
+    always_match = add_browser_capabilities({"timeouts": {"script": 10}})
```

Comment:
I wonder what our original idea was to have a custom value for `timeouts` set. Maybe it came in here via c&p? It shouldn't be needed.